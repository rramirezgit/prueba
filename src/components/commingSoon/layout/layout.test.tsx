import React from 'react'
import renderer from 'react-test-renderer'
import CommingSoonLayout from '.'

test('CommingSoonLayout snapshot', () => {
  const component = renderer.create(
    <CommingSoonLayout>
      <div>Test</div>
    </CommingSoonLayout>
  )
  const tree = component.toJSON()
  expect(tree).toMatchSnapshot()
})
