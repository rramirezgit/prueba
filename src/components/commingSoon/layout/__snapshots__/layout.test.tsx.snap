// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CommingSoonLayout snapshot 1`] = `
<div
  className="main"
>
  <nav
    className="nav"
  >
    <img
      alt="logo"
      data-nimg="1"
      decoding="async"
      height={40}
      loading="lazy"
      onError={[Function]}
      onLoad={[Function]}
      src="/_next/image?url=%2Fimg.jpg&w=96&q=75"
      srcSet="/_next/image?url=%2Fimg.jpg&w=48&q=75 1x, /_next/image?url=%2Fimg.jpg&w=96&q=75 2x"
      style={
        {
          "color": "transparent",
        }
      }
      width={40}
    />
  </nav>
  <div
    className="body"
  >
    <div>
      Test
    </div>
  </div>
</div>
`;
